src/main/resources/avro/referencemethods.avdl

```diff
@@ -12,19 +12,16 @@ as JSON.
 */
 record SearchReferenceSetsRequest {
   /**
-  If nonempty, return the reference sets which match any of the given
-  `md5checksum`s. See `ReferenceSet::md5checksum` for details.
+  If not null, return the reference sets which match the given
+  `md5checksum`. See `ReferenceSet::md5checksum` for details.
   */
-  array<string> md5checksums = [];
+  union { null, string } md5checksum = null;
 
   /**
-  If nonempty, return reference sets for which the accession
-  matches this string. Best to give a version number (e.g. `GCF_000001405.26`).
-  If only the main accession number is given then all records with
-  that main accession will be returned, whichever version.
-  Note that different versions will have different sequences.
+  If not null, return reference sets for which the accession
+  matches this string.
```

Comment:
Good point @dglazer --- we're quite specific about this elsewhere, so we should make this string matching description uniform. I've pushed a commit addressing this (I'll squash the commits after if/when we've agreed they should go in).